shared/overview/2.x.md

```diff
@@ -98,7 +98,7 @@ The first partition, `resin-boot`, host boot important bits according to each bo
 
 ## OS Yocto Composition
 
-The OS is composed of multiple Yocto layers. The [Yocto Project](https://www.yoctoproject.org/) build system uses these layers to compile {{ $names.os.lower }} for the various [supported platforms](./supportedboards.md).
+The OS is composed of multiple Yocto layers. The [Yocto Project](https://www.yoctoproject.org/) build system uses these layers to compile {{ $names.os.lower }} for the various [supported platforms](/reference/hardware/devices/).
```

Comment:
There is a complication here. I've come across the **supported platforms** broken link in two different "sites":
* https://docs.resin.io/reference/OS/overview/2.x/#os-yocto-composition
* https://resinos.io/docs/architecture/

The link is broken in both sites, but the fix is different for each:
* docs.resin.io → `'/reference/hardware/devices/'`
* resinos.io → `'/docs/supportedboards/'`

This pull request, as it is, only fixes the problem for **docs.resin.io**. Is the right solution to do something clever with dollar variables?

Reply:
@pdcastro link for the one on resinos.io should be addressed here: https://github.com/resin-os/resinos/tree/master/pages/docs in the future we hope to unify this documentation, but no work has been started on that yet :/

Reply:
Thanks for reviewing my first pull request, @shaunmulligan. :-)
I see -- there is a copy of the documentation in the `resin-os/resinos` repository. Looking at the **supported platforms** link in that repo, it was interesting to find that the link is correct when browsing the docs as Markdown, but broken when browsing them as HTML:
* OK in the Markdown view: https://github.com/resin-os/resinos/blob/master/pages/docs/architecture.md
* Broken in the HTML view: https://resinos.io/docs/architecture/

Both views share the same link source: `'./supportedboards.md'`. I think what happens is that `'./'` means "same folder" under Markdown, but "subfolder" under HTML. In other words, changing the link to `'../'` would fix the HTML view, but then break the Markdown view.
My proposed course of action for now is to leave the `resin-os/resinos` repo alone (i.e. ignore it), because it is not a priority and no one raised an issue specifically for it. :-)

Reply:
@pdcastro sounds good to me, can you create an issue on resin-os/resinos for this so at somepoint we get it fixed up :)

Reply:
Actioned as discussed in Flowdock.